macos-guest-virtualbox.sh

```diff
@@ -673,6 +678,10 @@ VBoxManage setextradata "${vm_name}" \
   "ourhardworkbythesewordsguardedpleasedontsteal(c)AppleComputerInc"
 VBoxManage setextradata "${vm_name}" \
  "VBoxInternal/Devices/smc/0/Config/GetKeyFromRealSMC" 0
+if [ ${cpu_profile_on} = true ] ; then
```

Comment:
The `--cpu-profile "${cpu_profile}` option should be added to the existing `modifyvm` command when configuring the virtual machine.